### PR TITLE
set the version to 'dev-metadata' in integration tests

### PR DESCRIPTION
**Pull request by laverya**
also: use a new staging test app, and run integration tests on merge

### CODE REVIEW

File: .github/workflows/pull-request.yaml [hunk 1]
```diff
@@ -71,3 +62,38 @@ jobs:
       run: |
         export EC_VERSION="dev-metadata"
         ./scripts/cache-files.sh
+
+    - name: Cache files for integration test
```
Comment (ricardomaraschini) [inline .github/workflows/pull-request.yaml:66]: Why is this step duplicated ?

File: e2e/install_test.go [hunk 2]
```diff
@@ -70,6 +70,15 @@ func TestSingleNodeInstallation(t *testing.T) {
...
 	}
+	t.Log("checking installation state")
+	line = []string{"check-installation-state.sh"}
+	stdout, stderr, err = RunCommandOnNode(t, tc, 0, line)
+	if err != nil {
+		t.Log("stdout:", stdout)
+		t.Log("stderr:", stderr)
+		t.Fatalf("fail to check installation state: %v", err)
+	}
+	t.Log("installation state:", stdout)
```
Comment (ricardomaraschini) [inline e2e/install_test.go:81]: Is this just so we can see the output after the test run ?

File: .github/workflows/pull-request.yaml (hunk 1 above)
Comment (laverya) [reply .github/workflows/pull-request.yaml:66]: Because it's cashing to two different places

File: e2e/install_test.go (hunk 2 above)
Comment (laverya) [reply e2e/install_test.go:81]: Yes - I would prefer to have a check for the actual installation state, but it isn't failing yet 😆

**Review by ricardomaraschini — APPROVED** [review]

**Pull request merged**